keyboards/late9/rev1/keymaps/default/keymap.c

```diff
@@ -0,0 +1,284 @@
+/* Copyright 2021 rookiebwoy
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// Layers declarations
+enum layers {
+    _BL
+};
+
+// Tap Dance declarations
+enum {
+    TD_PNCT,
+    TD_ABC,
+    TD_DEF,
+    TD_GHI,
+    TD_JKL,
+    TD_MNO,
+    TD_PQRS,
+    TD_TUV,
+    TD_WXYZ,
+    TD_SPACE,
+    TD_UPRCS,
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+/* Keymap _BL: Base Layer
+ * ,-----------------------.
+ * |ENTER  | ↑     | BSPC  |
+ * |       |       |       |
+ * |-------|-------|-------|
+ * | <-    | ↓     | ->    |
+ * |       |       |       |
+ * |-------|-------|-------|
+ * | 1 .,' | 2     | 3     |
+ * |?!-:;/ | abc   | def   |
+ * |-------|-------|-------|
+ * | 4     | 5     | 6     |
+ * | ghi   | jkl   | mno   |
+ * |-------|-------|-------|
+ * | 7     | 8     | 9     |
+ * | pqrs  | tuv   | wxyz  |
+ * |-------|-------|-------|
+ * | *     | 0     | CAPS  |
+ * |       | space |       |
+ * `-----------------------'
+ */
+  [_BL] = LAYOUT(
+    KC_ENT,       KC_UP,        KC_BSPC,
+    KC_LEFT,      KC_DOWN,      KC_RIGHT,
+    TD(TD_PNCT),  TD(TD_ABC),   TD(TD_DEF),
+    TD(TD_GHI),   TD(TD_JKL),   TD(TD_MNO),
+    TD(TD_PQRS),  TD(TD_TUV),   TD(TD_WXYZ),
+    KC_PAST,      TD(TD_SPACE), TD(TD_UPRCS)
+  ),
+};
+
+// Tap Dance cycles
```

Comment:
Default keymap should be simple

Reply:
This is not complicated as it might seem (especially for you because it's your code LOL). It's always the same cycle for every key. The only one with different behavior is the # key.
I develop this keyboard to use this keymap, but if it's a problem I can make a simpler numpad as default, and this one as an alternative. In this scenario, I should leave in the readme.md instructions on how to build the default one? or can I explain the difference between the two keymaps? Let me know if this can be a solution. Thank you.

Reply:
Can I have an update about this pull request please? Thanks in advance.

Reply:
I'm still concerned about this.  The default keymap should be as simple and plain as possible.  

From the PR Checklist
```
* default (and via) keymaps should be “pristine”
  * bare minimum to be used as a “clean slate” for another user to develop their own user-specific keymap
  * standard layouts preferred in these keymaps, if possible
```

That said, as a second keymap would be more than fine.

Reply:
Default keymap is now a simple one.